lte/protos/session_manager.proto

```diff
@@ -540,3 +556,226 @@ service CentralSessionController {
+     SEMANTICALLY_INCORRECT_MESSAGE = 30;                          //actual value 95
+     INVALID_MANDATORY_INFORMATION = 31;                           //actual value 96
+     MESSAGE_TYPE_NON_EXISTENT_OR_NOT_IMPLEMENTED = 32;            //actual value 97
+     MESSAGE_TYPE_NOT_COMPATIBLE_WITH_THE_PROTOCOL_STATE = 33;     //actual value 98
+     INFORMATION_ELEMENT_NON_EXISTENT_OR_NOT_IMPLEMENTED = 34;     //actual value 99
+     CONDITIONAL_IE_ERROR = 35;                                    //actual value 100
+     MESSAGE_NOT_COMPATIBLE_WITH_THE_PROTOCOL_STATE = 36;          //actual value 101
+     PROTOCOL_ERROR_UNSPECIFIED = 37;
+     PTI_ALREADY_IN_USE = 38;                                      //actual value 111
+     OPERATION_SUCCESS = 40;//Response other than failure
+}
+//RequestType
+enum RequestType {
+   INITIAL_REQUEST = 0;
+   EXISTING_PDU_SESSION = 1;
+   INITIAL_EMERGENCY_REQUEST =2;
+   EXISTING_EMERGENCY_PDU_SESSION = 3;
+   MODIFICATION_REQUEST = 4;
+}
+//AccessType
+enum AccessType {
+  M_3GPP_ACCESS_3GPP = 0;
+  NON_3GPP_ACCESS = 1;
+}
+//DnnSelectionMode
+enum DnnSelectionMode {
+    VERIFIED = 0;
+    UE_DNN_NOT_VERIFIED = 1;
+    NW_DNN_NOT_VERIFIED = 2;
+}
+
+//RatType
+//Define enum TS-29.571 - section 5.4.3 structure Imported from session_manager.proto file.
+
+//RequestedQosRules and AuthorizedQosRules
+message QosRules {
+   uint32 qos_rule_identifier = 1;
+   bool dqr = 2;
+   uint32 number_of_packet_filters = 3;
+   repeated int32 packet_filter_identifier = 4;
+   uint32 qos_rule_precedence = 5;
+   bool segregation = 6;
+   uint32 qos_flow_identifier = 7;
+}
+
+message LadnServiceArea {
+   string dnn_value = 1;
+   repeated TrackingAreaIdentityList identity_list = 2;
+}
+message TrackingAreaIdentityList {
+   repeated TypeOfList type_of_list = 1;
+   repeated int32 number_of_elements = 2;
+   string MCC=3;
+   string MNC=4;
+   string TAC= 5;          // tac is havaing length 8 bytes,MCC is 3 bytes, MNC is 3 bytes
+}
+enum TypeOfList  {
+   LIST_OF_TACS_BELONGING_TO_ONE_PLMN_WITH_NON_CONSECUTIVE_TAC_VALUES = 0;
+   LIST_OF_TACS_BELONGING_TO_ONE_PLMN_WITH_CONSECUTIVE_TAC_VALUES = 1;
+   LIST_OF_TAIS_BELONGING_TO_DIFFERENT_PLMNS = 2;
+}
+enum priorityaccess  {
+   High = 0;
+   Normal = 1;
+   Low = 2;
+}
+
+enum SMSessionFSMState {
+        CREATING_0 = 0;
+        CREATE_1 = 1;
+        ACTIVE_2 = 2;
+        INACTIVE_3 =3;
+        RELEASED_4 = 4;
+}
+
+
+//PDU session Establishment from AMF to SMF
+//SET MESSSAGE
+message M5GSMSessionContext {
+  bytes pdu_session_id = 1;   //One byte
+  RequestType rquest_type = 2;        //Define ENUM 24-501 - 9.11.3.47 . 1 byte. i.e. initial requerst, existing pdusession etc...
+  RedirectServer pdu_address = 3; //IP address. Imported from session_manager.proto file
+  priorityaccess priority_access = 4;
+  AccessType access_type = 5;     //Define enum TS-29.571 - section 5.4.3
+  SubscriberID sid = 25;
```

Comment:
This is included in CommonContext, I think

Reply:
please don't use `sid`. Use `subscriber_id`. This is unfortunately needed to make sure it is not confused with `session_id`

Reply:
> This is included in CommonContext, I think

It had already been taken care.

Reply:
> please don't use `sid`. Use `subscriber_id`. This is unfortunately needed to make sure it is not confused with `session_id`

sid is already being used in LTE code. It had been reused in 5G and put inside common section.

Reply:
@uri200 , I hope I can resolve this comment.

Reply:
Should we remove this then? @sanjay-ACL (wasn't sure if this commit was updated or not)